src/server/api/api.ts

```diff
@@ -67,3 +128,24 @@ function makeMove(move: { from: string; to: string }) {
 function processMove(from: Square, to: Square): Command {
   throw new Error("Function not implemented");
 }
+
+function manualMove(robotId: number, leftPower: number, rightPower: number): boolean {
+  if (!(tcpServer.getConnectedIDs().includes(robotId.toString()))) {

```

Comment:
This function uses both `ID` and `Id`. I personally vote in favor of standardizing around `Id` - my understanding is that's what google recommends as well.